civictechprojects/static/css/partials/_FavoriteToggle.scss

```
@@ -10,3 +9,15 @@
     font-size: 24px;
   }
 }
+
+.favorited {
+  color: $color-favorite;
+}
+
+.unfavorited {
+  color: $color-grey-unselected;
+}
+
+.unfavorited:hover {
```

```suggestion
.favorite-toggle:hover .unfavorited {
```

This should make the hover color change apply on the entire click target and not just the icon specifically.